### ISSUE TITLE
Value of filter criterion for field must be of type string and should be enclosed in quotes

### ISSUE THREAD

**Issue by axsweet** [roblesterjr04/EloquentSalesForce#68]
Laravel 7.30.4
rob-lester-jr04/eloquent-sales-force    2.7.5            

Anyone have any ideas here:

I can force this to be in quotes but I get no results.

When I add like and a % after the search I get the result.

Simple first() or get() calls are not working on some fields. Not sure if its a versioning or method piece.

  [
    {
        "message": "\nProductCode from Product2 where ProductCode = 3999\n                                ^\nERROR at Row:1:Column:44\nvalue of filter criterion for field 'ProductCode' must be of type string and should be enclosed in quotes",
        "errorCode": "INVALID_FIELD"
    }
] (SQL: select `Id`, `ProductCode` from `Product2` where `ProductCode` = 3999)

**Comment by axsweet** [comment]
OK so the first one was a number, the rest are actual strings so seems to be easier to handle when always a string, the entire rest of the list is strings so I handled accordingly.

**Comment by Wit3** [comment]
Same error.

**Comment by axsweet** [comment]
@roblesterjr04 any ideas here, I keep running into these issues and end up going in circles.  I did pull the latest updates and of forrest.  Would love a fix of some kind

**Comment by axsweet** [comment]
Yes both are part of a default.  Someone else suggested its part of the underlying forrest call in another group, I tried adding quotes myself also same result, im sure you do also, I had tried many things so that why I arrived at the like statement work around.

**Comment by roblesterjr04** [comment]
So when you use `first()` on a query, it is modifying the statement to include `limit 1` - what you could do as a workaround is use `get()` instead, and then apon results, call `first()` on the collection. But this tells me i need to look at what the builder is doing when it adds the `limit` to the end of the statement.
